2024-02-02 20:01:05,960:DEBUG:certbot._internal.main:certbot version: 2.8.0
2024-02-02 20:01:05,960:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2024-02-02 20:01:05,960:DEBUG:certbot._internal.main:Arguments: ['--webroot', '--webroot-path=/usr/share/nginx/html/letsencrypt', '--email', 'desarrollo.jadalsoftware@gmail.com', '--agree-tos', '--no-eff-email', '-d', 'migracion.jadal.pe']
2024-02-02 20:01:05,961:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2024-02-02 20:01:06,005:DEBUG:certbot._internal.log:Root logging level set at 30
2024-02-02 20:01:06,007:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2024-02-02 20:01:06,007:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A seperate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7fc534d24a30>
Prep: True
2024-02-02 20:01:06,007:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7fc534d24a30> and installer None
2024-02-02 20:01:06,007:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2024-02-02 20:01:06,175:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/1550272667', new_authzr_uri=None, terms_of_service=None), 9b507c65556e201cf96f44fb1f74216d, Meta(creation_dt=datetime.datetime(2024, 2, 2, 17, 5, 57, tzinfo=<UTC>), creation_host='ee38a32c6a85', register_to_eff=None))>
2024-02-02 20:01:06,186:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2024-02-02 20:01:06,188:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2024-02-02 20:01:06,482:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 752
2024-02-02 20:01:06,483:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 20:01:06 GMT
Content-Type: application/json
Content-Length: 752
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.3-September-21-2022.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-v02.api.letsencrypt.org/draft-ietf-acme-ari-02/renewalInfo/",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert",
  "yWopG-F3vcs": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417"
}
2024-02-02 20:01:06,483:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for migracion.jadal.pe
2024-02-02 20:01:06,489:DEBUG:acme.client:Requesting fresh nonce
2024-02-02 20:01:06,491:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2024-02-02 20:01:06,565:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2024-02-02 20:01:06,566:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 20:01:06 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 2ksh1KUsfE87BonFLCzp27PwANCKm1AOljHkIN4k2UHk-mqsGYk
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2024-02-02 20:01:06,566:DEBUG:acme.client:Storing nonce: 2ksh1KUsfE87BonFLCzp27PwANCKm1AOljHkIN4k2UHk-mqsGYk
2024-02-02 20:01:06,567:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "migracion.jadal.pe"\n    }\n  ]\n}'
2024-02-02 20:01:06,570:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MDI3MjY2NyIsICJub25jZSI6ICIya3NoMUtVc2ZFODdCb25GTEN6cDI3UHdBTkNLbTFBT2xqSGtJTjRrMlVIay1tcXNHWWsiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "p9Ltfc7TltoLPIinFZno7bvcxGDT_X6iOhNDZe5laO1gc3cScZg9YynJK_emQHQMMjbB55a334bHF4yqqcgeMUdxrz-PwSXTbFd7Ef6OtjP3FEoeAvZSCcjFxiYsz9nNjZPk_M0wHt1VtT1KFJBaUv-ro7DHpouqLItxs7LwqfcjrTyYfQ5jImzNPgOQxlhM6lONsO6GHylRQJjGK42pdoyc9Wms5JvKaJco2rTYIsWhfArWLmglccYs5yw-ZuzgSG_g_pmm4uwwBW3fgu9ljbO3dns-w-5-CyXrOOCTZy8BKz1dwsore0NFjHQAcsEfEoBDwiKatgAGvJXv-j87YQ",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogIm1pZ3JhY2lvbi5qYWRhbC5wZSIKICAgIH0KICBdCn0"
}
2024-02-02 20:01:06,753:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 344
2024-02-02 20:01:06,753:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Fri, 02 Feb 2024 20:01:06 GMT
Content-Type: application/json
Content-Length: 344
Connection: keep-alive
Boulder-Requester: 1550272667
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/1550272667/241359314387
Replay-Nonce: kRyagOpUbNkeu87c1iqFxi6OqcfZaFRbWwlP1GBXZK_RsUItQFw
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2024-02-09T20:01:06Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "migracion.jadal.pe"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz-v3/310890876117"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/1550272667/241359314387"
}
2024-02-02 20:01:06,753:DEBUG:acme.client:Storing nonce: kRyagOpUbNkeu87c1iqFxi6OqcfZaFRbWwlP1GBXZK_RsUItQFw
2024-02-02 20:01:06,754:DEBUG:acme.client:JWS payload:
b''
2024-02-02 20:01:06,757:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/310890876117:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MDI3MjY2NyIsICJub25jZSI6ICJrUnlhZ09wVWJOa2V1ODdjMWlxRnhpNk9xY2ZaYUZSYld3bFAxR0JYWktfUnNVSXRRRnciLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMxMDg5MDg3NjExNyJ9",
  "signature": "xCnUx22Ce2-d9OQw4aBTfB4qcfDEuWNvkr-rvCZy_Xc02nCOqPJhTUL0iLzTwZv8miVRmXugIhe4oHS7dKubkhvHujEd_-tAfFG-uT-jeZ2Ml7fI4FVwOyDXdkLlM5mxNLhvqXFyOfBQF9VyseWeak6j-7iBZ3z7lodbjLqBHe1tSKJrpRxeZ0NEm4KvhfGH-4QOJDiHNtenXErjzuQQYyhwhQ6GBObkpbbuKz06jrmvTStMX0og1pF-PkYthnWJFqxGHz_rKn8GGY0zdlhRWuRiyu8W1vehijcLd_js8eKr7kSw6nH1vcFN5B0trT8W6ZoI3_6uXSvcB5w-Wh18GA",
  "payload": ""
}
2024-02-02 20:01:06,834:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/310890876117 HTTP/1.1" 200 802
2024-02-02 20:01:06,834:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 20:01:06 GMT
Content-Type: application/json
Content-Length: 802
Connection: keep-alive
Boulder-Requester: 1550272667
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 2ksh1KUs4OtXSYXKiDKZVmXL4NPZjW_r-5FhLANSV-Dsuo0Dlfw
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "migracion.jadal.pe"
  },
  "status": "pending",
  "expires": "2024-02-09T20:01:06Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310890876117/fqJjfQ",
      "token": "1bRyPzvDb8tyaiCvwFX7OyH1fN7-P4H3QQcqfyXudro"
    },
    {
      "type": "dns-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310890876117/_Zifrw",
      "token": "1bRyPzvDb8tyaiCvwFX7OyH1fN7-P4H3QQcqfyXudro"
    },
    {
      "type": "tls-alpn-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310890876117/x_10VQ",
      "token": "1bRyPzvDb8tyaiCvwFX7OyH1fN7-P4H3QQcqfyXudro"
    }
  ]
}
2024-02-02 20:01:06,835:DEBUG:acme.client:Storing nonce: 2ksh1KUs4OtXSYXKiDKZVmXL4NPZjW_r-5FhLANSV-Dsuo0Dlfw
2024-02-02 20:01:06,835:INFO:certbot._internal.auth_handler:Performing the following challenges:
2024-02-02 20:01:06,835:INFO:certbot._internal.auth_handler:http-01 challenge for migracion.jadal.pe
2024-02-02 20:01:06,835:INFO:certbot._internal.plugins.webroot:Using the webroot path /usr/share/nginx/html/letsencrypt for all unmatched domains.
2024-02-02 20:01:06,836:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge
2024-02-02 20:01:06,838:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge/1bRyPzvDb8tyaiCvwFX7OyH1fN7-P4H3QQcqfyXudro
2024-02-02 20:01:06,838:DEBUG:acme.client:JWS payload:
b'{}'
2024-02-02 20:01:06,839:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall-v3/310890876117/fqJjfQ:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MDI3MjY2NyIsICJub25jZSI6ICIya3NoMUtVczRPdFhTWVhLaURLWlZtWEw0TlBaaldfci01RmhMQU5TVi1Ec3VvMERsZnciLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLXYzLzMxMDg5MDg3NjExNy9mcUpqZlEifQ",
  "signature": "eq8wZ8hA808AWYJB7GUKjG-GGi7fwUIY6Nw30iULnscf70WnM4YHyvcApOvNvES9qhIGGN6kmzIyeV31SGeR14LVp9xVvoTl-suHACseoIlt7PfT3gonX7Pmy7VG2C646pa8PLI6_LqH4zG88RyTyRCOvRSqQPUtFrJVBYeTOKunSl1LISCRgn1MnHvYSfOJwDbtIKOqYEkllaro1Ra5IiOB62mL5WGwmUL9pjTd7IOoV9JvlJ2Kk57dGrAIinJSZyJa1yNXhqpeeHmoqwdoep-clq_kM18eMuCiaLog9eBAB0onoiAiVUIlUaXfNznbWFhKzUm93GvH3U2P9fooNg",
  "payload": "e30"
}
2024-02-02 20:01:06,923:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall-v3/310890876117/fqJjfQ HTTP/1.1" 200 187
2024-02-02 20:01:06,923:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 20:01:06 GMT
Content-Type: application/json
Content-Length: 187
Connection: keep-alive
Boulder-Requester: 1550272667
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz-v3/310890876117>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall-v3/310890876117/fqJjfQ
Replay-Nonce: 2ksh1KUsj3AcuTLXXnceBdtwVo_ketPoED7jyaY91FZlC0F_uTM
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "status": "pending",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310890876117/fqJjfQ",
  "token": "1bRyPzvDb8tyaiCvwFX7OyH1fN7-P4H3QQcqfyXudro"
}
2024-02-02 20:01:06,924:DEBUG:acme.client:Storing nonce: 2ksh1KUsj3AcuTLXXnceBdtwVo_ketPoED7jyaY91FZlC0F_uTM
2024-02-02 20:01:06,924:INFO:certbot._internal.auth_handler:Waiting for verification...
2024-02-02 20:01:07,925:DEBUG:acme.client:JWS payload:
b''
2024-02-02 20:01:07,927:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/310890876117:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MDI3MjY2NyIsICJub25jZSI6ICIya3NoMUtVc2ozQWN1VExYWG5jZUJkdHdWb19rZXRQb0VEN2p5YVk5MUZabEMwRl91VE0iLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMxMDg5MDg3NjExNyJ9",
  "signature": "Fx4YnttPyyMwMJZioDjw8rqqVeb7-Klvl7FzybjU4wULvcIlQumUrTC2jwqo3OYQ2AWApTr9UUCnwXZNgm1stDqjr0DyXV-7CG9xGfoBJLT4SkF09kwvsC4CVqKbOzgK8PALRMBLaXnX5UP457zHr1M95X3N5O08l_N163-5Y2GEIadK0RKcwK7XFMkExzsiTw-f5Jnoy3swnVaX-kZ5HtxCF3nYlDb7NaR8lKftNyMwfSnZNIKeNa0suTGfU96tkPJvhEfP8Q81HRk7sF77V2K2TQUNpPqKdw-xbRyE6ju2HU7TAXpqx8B7Ahky-h1KyOACO3uyzYpfc7MDb_tM8g",
  "payload": ""
}
2024-02-02 20:01:08,004:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/310890876117 HTTP/1.1" 200 802
2024-02-02 20:01:08,005:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 20:01:07 GMT
Content-Type: application/json
Content-Length: 802
Connection: keep-alive
Boulder-Requester: 1550272667
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: kRyagOpUUQWelQIxkRrWKLQyKFWoGlRfxOzLESewfV7fYFzGWDY
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "migracion.jadal.pe"
  },
  "status": "pending",
  "expires": "2024-02-09T20:01:06Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310890876117/fqJjfQ",
      "token": "1bRyPzvDb8tyaiCvwFX7OyH1fN7-P4H3QQcqfyXudro"
    },
    {
      "type": "dns-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310890876117/_Zifrw",
      "token": "1bRyPzvDb8tyaiCvwFX7OyH1fN7-P4H3QQcqfyXudro"
    },
    {
      "type": "tls-alpn-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310890876117/x_10VQ",
      "token": "1bRyPzvDb8tyaiCvwFX7OyH1fN7-P4H3QQcqfyXudro"
    }
  ]
}
2024-02-02 20:01:08,005:DEBUG:acme.client:Storing nonce: kRyagOpUUQWelQIxkRrWKLQyKFWoGlRfxOzLESewfV7fYFzGWDY
2024-02-02 20:01:11,008:DEBUG:acme.client:JWS payload:
b''
2024-02-02 20:01:11,010:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/310890876117:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MDI3MjY2NyIsICJub25jZSI6ICJrUnlhZ09wVVVRV2VsUUl4a1JyV0tMUXlLRldvR2xSZnhPekxFU2V3ZlY3ZllGekdXRFkiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMxMDg5MDg3NjExNyJ9",
  "signature": "fVlHCn-YFt0Yt20-Xb68kPQO-VLYT5CoKGWam-X5pd_CoZCGM9o7XY6RJKk0tQEzLei447WXngMToJ3XU7lEmRpKkutspvloF0T0wyi4nfhkD5ysGETmUItyFai8aTMWXK6nZ7JH3cN9YHFPMozp5lSeHHkvNHGuSLL-1eGzdv9N9sXrXGRJcMyw6fovHMw9BsvC3aWP6f8VI70M98XceGqrrduc4c7fkjYdUxl41xui_hMVEQlnaSCHXcwRfRpzuC24ADQ4OWvtSLGF2ht-N2eiXpWegbzATBGcEHb8nEWuvOx8o30R10I8KNo03h-wC25xYc9bdUYpuEUvaJBeNA",
  "payload": ""
}
2024-02-02 20:01:11,088:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/310890876117 HTTP/1.1" 200 1354
2024-02-02 20:01:11,088:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 20:01:11 GMT
Content-Type: application/json
Content-Length: 1354
Connection: keep-alive
Boulder-Requester: 1550272667
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 2ksh1KUsjcWflm2hmRGBFLtyW3l5B0zHSFQt7nemqiFuq2ohZ3g
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "migracion.jadal.pe"
  },
  "status": "invalid",
  "expires": "2024-02-09T20:01:06Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "invalid",
      "error": {
        "type": "urn:ietf:params:acme:error:unauthorized",
        "detail": "3.85.216.85: Invalid response from https://migracion.jadal.pe/.well-known/acme-challenge/1bRyPzvDb8tyaiCvwFX7OyH1fN7-P4H3QQcqfyXudro: 404",
        "status": 403
      },
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310890876117/fqJjfQ",
      "token": "1bRyPzvDb8tyaiCvwFX7OyH1fN7-P4H3QQcqfyXudro",
      "validationRecord": [
        {
          "url": "http://migracion.jadal.pe/.well-known/acme-challenge/1bRyPzvDb8tyaiCvwFX7OyH1fN7-P4H3QQcqfyXudro",
          "hostname": "migracion.jadal.pe",
          "port": "80",
          "addressesResolved": [
            "3.85.216.85"
          ],
          "addressUsed": "3.85.216.85"
        },
        {
          "url": "https://migracion.jadal.pe/.well-known/acme-challenge/1bRyPzvDb8tyaiCvwFX7OyH1fN7-P4H3QQcqfyXudro",
          "hostname": "migracion.jadal.pe",
          "port": "443",
          "addressesResolved": [
            "3.85.216.85"
          ],
          "addressUsed": "3.85.216.85"
        }
      ],
      "validated": "2024-02-02T20:01:06Z"
    }
  ]
}
2024-02-02 20:01:11,089:DEBUG:acme.client:Storing nonce: 2ksh1KUsjcWflm2hmRGBFLtyW3l5B0zHSFQt7nemqiFuq2ohZ3g
2024-02-02 20:01:11,089:INFO:certbot._internal.auth_handler:Challenge failed for domain migracion.jadal.pe
2024-02-02 20:01:11,089:INFO:certbot._internal.auth_handler:http-01 challenge for migracion.jadal.pe
2024-02-02 20:01:11,089:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: migracion.jadal.pe
  Type:   unauthorized
  Detail: 3.85.216.85: Invalid response from https://migracion.jadal.pe/.well-known/acme-challenge/1bRyPzvDb8tyaiCvwFX7OyH1fN7-P4H3QQcqfyXudro: 404

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2024-02-02 20:01:11,091:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2024-02-02 20:01:11,092:DEBUG:certbot._internal.error_handler:Calling registered functions
2024-02-02 20:01:11,092:INFO:certbot._internal.auth_handler:Cleaning up challenges
2024-02-02 20:01:11,092:DEBUG:certbot._internal.plugins.webroot:Removing /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge/1bRyPzvDb8tyaiCvwFX7OyH1fN7-P4H3QQcqfyXudro
2024-02-02 20:01:11,092:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2024-02-02 20:01:11,092:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
  File "/opt/certbot/src/certbot/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1869, in main
    return config.func(config, plugins)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1600, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 143, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 517, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 428, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 496, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2024-02-02 20:01:11,094:ERROR:certbot._internal.log:Some challenges have failed.
